src/main/java/org/broadinstitute/hellbender/tools/walkers/CountVariants.java

```diff
@@ -9,15 +9,33 @@
 import org.broadinstitute.hellbender.engine.ReferenceContext;
 import org.broadinstitute.hellbender.engine.VariantWalker;
 
+/**
+ *
+ * Count variants in a VCF file.
```

Comment:
--> Counts variant records in a VCF file, regardless of filter status. The tool gives the count at end of the standard out.